code/WorkInProgress/blueprints.dm

```diff
@@ -436,9 +436,7 @@
 
 	var/static/savefile/save = new/savefile("data/blueprints.dat")
 
-	afterattack(atom/target as mob|obj|turf, mob/user as mob)
-		if(GET_DIST(src,target) > 2) return
```

Comment:
seems you removed the distance check, is that intended?

Reply:
should probably retain *some* distance check (7 or 10), to lock it to stuff on-screen (camera shenanigans etc.)

Reply:
I just copied Floor and Wall Designer's design (obj/item/room_planner), which also has no distance check, and seemingly no problems

Reply:
those should be limited to on-screen too 😱